cmd/traffic_layout/engine.cc

```diff
@@ -164,7 +163,7 @@ bool
 RunrootEngine::runroot_parse()
 {
   int i = 1;
```

Comment:
instead of casting... just change the type to unsigned

Reply:
I meant, why not for (unsigned i=1; ...